hs_core/management/commands/refresh_resource_cached_metadata.py

```diff
@@ -0,0 +1,144 @@
+"""
+Management command to refresh the cached metadata for a specific resource or all resources.
```

Comment:
As part of the deployment, we have to run this management command to populate the `cached_metadata` json field for all resources.

Reply:
Can you add this as a migration step after `0085_alter_baseresource_creator_and_more` below?

Reply:
I have a added new custom migration script to run this management command.